Header header   
bool action_source   #0 is navigation, 1 is RL policy
int32 control_action #The direction of the control action
float32 acc      
